### DISCUSSION TITLE
graphscope.framework.errors.ConnectionError: connect   coordinator  timeout,grpc  connect    failed

### DISCUSSION THREAD

**Discussion by yzjg** [alibaba/GraphScope#485] [Q&A]
Describe the bug
according to the offical document to configure the argument,then the bug throw out,cann't run normally

Environment (please complete the following information):

GraphScope version: 0.4.0
OS: centos7.7
Kubernetes Version 1.16.4
![image](https://user-images.githubusercontent.com/25189806/124694749-77f7ca80-df14-11eb-84f4-2825042d3166.png)
![image](https://user-images.githubusercontent.com/25189806/124694764-7e864200-df14-11eb-8e20-6576c9e9b9dc.png)
![image](https://user-images.githubusercontent.com/25189806/124694774-83e38c80-df14-11eb-88fa-e998ef00deb1.png)

**Comment by lidongze0629** [comment]
Hi, @yzjg. Thanks for your interest in GraphScope. From your log above,  the client try to connect to the GraphScope service with address `10.0.20.102:31041`, but timeout.

> `Connect to GraphScope session with address: 10.0.20.102:31041`

So, May I know how you deploy your Kubernetes cluster and GraphScope . GraphScope looks like deployed with Helm, and you can double-check whether the client can access the address `telnet 10.0.20.102 31041`

**Reply by yzjg** [reply]
![image](https://user-images.githubusercontent.com/25189806/124712857-66bdb680-df32-11eb-8958-c0ea2e159270.png)
It  is  ok

**Reply by yzjg** [reply]
i  have  a  question,I  can  run  on    python  command  line  successfully,but  when  i  pasted  the  same  code    on  the   jupyter   notebook,then  there   throw   a   grpc    timeout   exception
 to  the  jupyter notebook,has   extra   set  some  configure

**Reply by sighingnow** [reply]
Do your jupyter notebook and your python command line share the same environment?

i.e., are they run on the same machine?
